### ISSUE TITLE
[Bug]: Something seems wrong in 'get_chebyshev_scalarization'

### ISSUE THREAD

**Issue by qlchen2117** [pytorch/botorch#3110]
### What happened?

The original code of `botorch.utils.multi_objective.scalarization.get_chebyshev_scalarization` (Line 107) is
```
Y_normalized[..., minimize] = Y_normalized[..., minimize] - 1
```
It seems it should be
```
Y_normalized[..., minimize] = Y_normalized[..., minimize] * -1
```

### Please provide a minimal, reproducible example of the unexpected behavior.

`minimize = weights < 0` is a boolean mask indicating minimization. Hence, values of `Y_normalized[..., minimize]` should be multiplied by -1 rather than added to -1.

### Please paste any relevant traceback/logs produced by the example provided.

```shell

```

### BoTorch Version

0.16.1

### Pull Request

None

### Code of Conduct

- [x] I agree to follow BoTorch's Code of Conduct

**Comment by Balandat** [comment]
If you look at the code around it, you'll see that the subtraction of `1` here is to shift the values from `[0, 1]` to `[-1, 0]`. Multiplication with `-1` is done subsequently in the return statement, so this should be correct. 

https://github.com/meta-pytorch/botorch/blob/abb09b6db52e319514ca643ffc5f2af064dec7f8/botorch/utils/multi_objective/scalarization.py#L102-L110

**Comment by Balandat** [comment]
Lmk if that clarifies things - closing this for now, feel free to reopen if you have additional concerns / questions.
